src/pages/EditSplitBillPage.js

```diff
@@ -0,0 +1,170 @@
+    const transaction = TransactionUtils.getLinkedTransaction(reportAction);
+
+    let draftSplitTransaction = draftSplitTransactions[`${ONYXKEYS.COLLECTION.DRAFT_SPLIT_TRANSACTION}${transaction.transactionID}`];
+    if (!draftSplitTransaction) {
+        IOU.setDraftSplitTransaction(transaction.transactionID);
+    }
+
+    const {
+        amount: transactionAmount,
+        currency: transactionCurrency,
+        comment: transactionDescription,
+        merchant: transactionMerchant,
+        created: transactionCreated,
+    } = draftSplitTransaction ? ReportUtils.getTransactionDetails(draftSplitTransaction) : ReportUtils.getTransactionDetails(transaction);
+
+    const defaultCurrency = lodashGet(route, 'params.currency', '') || transactionCurrency;
+
+    function setDraftSplitTransaction(transactionChanges) {
+        IOU.setDraftSplitTransaction(transaction.transactionID, transactionChanges);
+        Navigation.navigate(ROUTES.SPLIT_BILL_DETAILS.getRoute(report.reportID, reportAction.reportActionID));
+    }
+
+    if (fieldToEdit === CONST.EDIT_REQUEST_FIELD.DESCRIPTION) {
+        return (
+            <EditRequestDescriptionPage
+                defaultDescription={transactionDescription}
+                onSubmit={(transactionChanges) => {
+                    if (transactionChanges.comment === transactionDescription) {
+                        Navigation.dismissModal();
+                        return;
+                    }
+                    setDraftSplitTransaction({
+                        comment: transactionChanges.comment,
+                    });
+                }}
+            />
+        );
+    }
+
+    if (fieldToEdit === CONST.EDIT_REQUEST_FIELD.DATE) {
+        return (
+            <EditRequestCreatedPage
+                defaultCreated={transactionCreated}
+                defaultAmount={transactionAmount}
+                reportID={report.reportID}
+                onSubmit={(transactionChanges) => {
+                    if (transactionChanges.created === transactionCreated) {
+                        Navigation.dismissModal();
+                        return;
+                    }
+                    setDraftSplitTransaction({
+                        created: transactionChanges.created,
+                    });
+                }}
+            />
+        );
+    }
+
+    if (fieldToEdit === CONST.EDIT_REQUEST_FIELD.AMOUNT) {
+        return (
+            <EditRequestAmountPage
+                defaultAmount={transactionAmount}
+                defaultCurrency={defaultCurrency}
+                reportID={report.reportID}
+                isEdittingSplitBill
+                onSubmit={(transactionChanges) => {
+                    const amount = CurrencyUtils.convertToBackendAmount(Number.parseFloat(transactionChanges));
+
+                    if (amount === transactionAmount && transactionCurrency === defaultCurrency) {
+                        Navigation.goBack();
```

Comment:
```suggestion
                        Navigation.dismissModal();
```

I think you meant dismissModal, right?

Reply:
No, we need to go back to the split details page, I'll change this to take the user back specifically to the route of the details in case of going back, in case they enter this page from a URL directly.